adapters/appnexus/appnexus.go

```diff
@@ -42,19 +42,24 @@ type KeyVal struct {
 }
 
 type appnexusParams struct {
-	PlacementId       int      `json:"placementId"`
-	InvCode           string   `json:"invCode"`
-	Member            string   `json:"member"`
-	Keywords          []KeyVal `json:"keywords"`
-	TrafficSourceCode string   `json:"trafficSourceCode"`
-	Reserve           float64  `json:"reserve"`
-	Position          string   `json:"position"`
+	LegacyPlacementId       int      `json:"placementId"`
+	LegacyInvCode           string   `json:"invCode"`
+	LegacyTrafficSourceCode string   `json:"trafficSourceCode"`
+	PlacementId             int      `json:"placement_id"`
+	InvCode                 string   `json:"inv_code"`
+	Member                  string   `json:"member"`
+	Keywords                []KeyVal `json:"keywords"`
+	TrafficSourceCode       string   `json:"traffic_source_code"`
+	Reserve                 float64  `json:"reserve"`
+	Position                string   `json:"position"`
+	UsePmtRule              *bool    `json:"use_pmt_rule"`
```

Comment:
Why a pointer?

Reply:
Because we need to track three values: "true", "false", and "not present". When we marshal the parameter and the pointer is null, we can `omitempty` and pass it along as not existing, the same way we got it. If it was not a pointer, a missing `use_pmt_rule` would be forced to be explicitly set to `false` when we marshal it again to send on to Impbus.

Reply:
Makes sense, thanks for the context!